test/cypress/integration/consumer-tools/ask-cfpb/answer-helpers.cy.js

```diff
@@ -3,6 +3,14 @@ export class AskCfpbAnswerPage {
     cy.visit('/ask-cfpb/').get('.ask-categories article li a').first().click();
   }
 
+  getSummaryContentLink() {
```

Comment:
Upon reading these tests, without recalling the specific way our collapsing happens on Ask pages, I was initially confused by this definition of a "summary content link" - I confused this with the "summary button" which is used to expand the content on mobile. Am I right that what this really refers to is "the arbitrary first link within the article content"? If so, maybe a more straightforward name, like `getFirstLinkInArticle`?

Reply:
I went with `getFirstLinkInSummary`, as the first link in the article may not be within the summary component. For example, https://www.consumerfinance.gov/ask-cfpb/what-should-i-do-when-a-debt-collector-contacts-me-en-1695/ has a first link in the notification.

Reply:
https://github.com/cfpb/consumerfinance.gov/pull/7551/commits/4d0d8b65eca7664616ed7aaf87d65c153828b804